setup.cfg

```diff
@@ -1,16 +1,24 @@
+# Generated from:
+# https://github.com/plone/meta/tree/master/config/default
+[bdist_wheel]
+universal = 0
```

Comment:
We *do* need it for a different reason.  When `zest.releaser` sees a `bdist_wheel` section, it creates a wheel next to the usual source dist.
Alternative is to use a section `zest.releaser` with `create-wheel = yes`.
See https://zestreleaser.readthedocs.io/en/latest/uploading.html#uploading-wheels

We could look into changing this to always build a wheel, when the `wheel` package is available.  But for now the section is still needed.

Reply:
Isn't it that we always want wheels theses days?

Reply:
I created an issue for this:
https://github.com/zestsoftware/zest.releaser/issues/406

We *do* want to keep the current config in all packages for now, at least until a final version of zest.releaser has been out for a couple of months, otherwise individuals doing Plone package releases may fail to create the wheels if they don't use the latest version.